### ISSUE TITLE
cannot provision vagrant box (prerequisites.sh not found & git debs not found)

### ISSUE THREAD

**Issue by rkuhn** [TechEmpower/FrameworkBenchmarks#1806]
~~~
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Importing base box 'ubuntu/trusty64'...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'ubuntu/trusty64' is up to date...
==> default: Setting the name of the VM: vagrant-development_default_1450339706883_4931
==> default: Clearing any previously set forwarded ports...
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: Forwarding ports...
    default: 8080 => 28080 (adapter 1)
    default: 22 => 2222 (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: Warning: Connection timeout. Retrying...
    default: 
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default: 
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if it's present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
    default: The guest additions on this VM do not match the installed version of
    default: VirtualBox! In most cases this is fine, but in rare cases it can
    default: prevent things such as shared folders from working properly. If you see
    default: shared folder errors, please make sure the guest additions within the
    default: virtual machine match the version of VirtualBox you have installed on
    default: your host and reload your VM.
    default: 
    default: Guest Additions Version: 4.3.34
    default: VirtualBox Version: 5.0
==> default: Setting hostname...
==> default: Configuring and enabling network interfaces...
==> default: Mounting shared folders...
    default: /vagrant => /Users/rkuhn/comp/Techempower/FrameworkBenchmarks/deployment/vagrant-development
    default: /FrameworkBenchmarks => /Users/rkuhn/comp/Techempower/FrameworkBenchmarks
==> default: Running provisioner: file...
==> default: Running provisioner: shell...
    default: Running: /var/folders/y8/x_cnylv929v51sxyk3svdqbr0000gn/T/vagrant-shell20151217-20978-yvorz.sh
==> default: Setting up convenience hosts entries
==> default: 127.0.0.1 TFB-database
==> default: 127.0.0.1 TFB-client
==> default: 127.0.0.1 TFB-server
==> default: Adding user `testrunner' ...
==> default: Adding new group `testrunner' (1002) ...
==> default: Adding new user `testrunner' (1002) with group `testrunner' ...
==> default: Creating home directory `/home/testrunner' ...
==> default: Copying files from `/etc/skel' ...
==> default: Installing prerequisites
==> default: /tmp/vagrant-shell: line 100: toolset/setup/linux/prerequisites.sh: No such file or directory
==> default: Reading package lists...
==> default: Building dependency tree...
==> default: Reading state information...
==> default: The following extra packages will be installed:
==> default:   git-man liberror-perl
==> default: Suggested packages:
==> default:   git-daemon-run git-daemon-sysvinit git-doc git-el git-email git-gui gitk
==> default:   gitweb git-arch git-bzr git-cvs git-mediawiki git-svn
==> default: The following NEW packages will be installed:
==> default:   git git-man liberror-perl
==> default: 0 upgraded, 3 newly installed, 0 to remove and 0 not upgraded.
==> default: Need to get 3,346 kB of archives.
==> default: After this operation, 21.6 MB of additional disk space will be used.
==> default: Get:1 http://archive.ubuntu.com/ubuntu/ trusty/main liberror-perl all 0.17-1.1 [21.1 kB]
==> default: Err http://archive.ubuntu.com/ubuntu/ trusty-updates/main git-man all 1:1.9.1-1ubuntu0.1
==> default:   404  Not Found [IP: 91.189.91.13 80]
==> default: Err http://security.ubuntu.com/ubuntu/ trusty-security/main git-man all 1:1.9.1-1ubuntu0.1
==> default:   404  Not Found [IP: 91.189.91.14 80]
==> default: Err http://security.ubuntu.com/ubuntu/ trusty-security/main git amd64 1:1.9.1-1ubuntu0.1
==> default:   404  Not Found [IP: 91.189.91.14 80]
==> default: Fetched 21.1 kB in 0s (23.2 kB/s)
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/g/git/git-man_1.9.1-1ubuntu0.1_all.deb  404  Not Found [IP: 91.189.91.14 80]
==> default: E
==> default: : 
==> default: Failed to fetch http://security.ubuntu.com/ubuntu/pool/main/g/git/git_1.9.1-1ubuntu0.1_amd64.deb  404  Not Found [IP: 91.189.91.14 80]
==> default: E
==> default: : 
==> default: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
==> default: Cloning project from TechEmpower/FrameworkBenchmarks master
==> default: /tmp/vagrant-shell: line 123: git: command not found
==> default: Setting up SSH access to localhost
==> default: Generating public/private rsa key pair.
==> default: Your identification has been saved in /home/vagrant/.ssh/id_rsa.
==> default: Your public key has been saved in /home/vagrant/.ssh/id_rsa.pub.
==> default: The key fingerprint is:
==> default: 3f:7e:c7:e5:60:d7:0a:40:06:f7:9e:d3:52:a9:5f:e6 vagrant@TFB-all
==> default: The key's randomart image is:
==> default: +--[ RSA 2048]----+
==> default: |        ...      |
==> default: |         .o.   . |
==> default: |         o  . o  |
==> default: |          .. =   |
==> default: |        S  .* . +|
==> default: |         .  .+o++|
==> default: |          o  +.*E|
==> default: |         . .. + .|
==> default: |          .. .   |
==> default: +-----------------+
==> default: Generating public/private rsa key pair.
==> default: Your identification has been saved in /home/testrunner/.ssh/id_rsa.
==> default: Your public key has been saved in /home/testrunner/.ssh/id_rsa.pub.
==> default: The key fingerprint is:
==> default: 3a:d3:2a:8f:bb:75:bf:d0:40:c9:0e:de:cc:db:f0:0b testrunner@TFB-all
==> default: The key's randomart image is:
==> default: +--[ RSA 2048]----+
==> default: |                 |
==> default: |       . .       |
==> default: |      . +        |
==> default: |     . B         |
==> default: |      . S        |
==> default: |       o B       |
==> default: |      = E +      |
==> default: |    .o = + .     |
==> default: |    +=o   +.     |
==> default: +-----------------+
==> default: cat: 
==> default: /home/vagrant/.ssh/authorized_keys
==> default: : Permission denied
==> default: testrunner ALL=(ALL:ALL) NOPASSWD: ALL
==> default: Installing all software
==> default: /tmp/vagrant-shell: line 169: cd: /home/vagrant/FrameworkBenchmarks: No such file or directory
==> default: /tmp/vagrant-shell: line 170: toolset/run-tests.py: No such file or directory
==> default: Setting up welcome message
~~~

**Comment by msmith-techempower** [comment]
@nbrady-techempower Assigning this to you.

I think the problem is [here](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/deployment/vagrant-common/bootstrap.sh#L100); that line should probably be updated to:
`source ~/FrameworkBenchmarks/toolset/setup/linux/prerequisites.sh`

**Comment by nbrady-techempower** [comment]
Got it, merging after CI runs. #1807

**Comment by msmith-techempower** [comment]
Travis-CI cannot help test this, so I have canceled the build and merged in myself. ^_^

**Comment by nbrady-techempower** [comment]
I believe this may be a security issue just found with ubuntu and git and files being moved around. http://www.ubuntu.com/usn/ I will keep an eye on this though.

**Comment by nbrady-techempower** [comment]
@rkuhn this should be solved now per #1808 - let me know if you encounter any other problems

**Comment by rkuhn** [comment]
Thanks for the quick fix, it works now!
